source/common/http/http1/codec_impl.cc

```diff
@@ -459,6 +459,7 @@ ConnectionImpl::ConnectionImpl(Network::Connection& connection, CodecStats& stat
       max_headers_kb_(max_headers_kb), max_headers_count_(max_headers_count) {
   output_buffer_.setWatermarks(connection.bufferLimit());
   http_parser_init(&parser_, type);
+  parser_.allow_chunked_length = 1;
```

Comment:
What about the case of responses?  It seems like the code to respect the config is specific to the server side, but the option to allow_chunked_length is set unconditionally on the base class which is shared across request and response processing.

I only see new tests for the request case.  Does the response case have appropriate coverage?

Reply:
+1.

Reply:
Ah yes good catch. We need to do this for responses also and either raise a protocol error or do the translation.

Reply:
Added response checks, still working on tests.